### PR TITLE
feat: add create_webhook in forum & voice channels

### PR DESCRIPTION
**Pull request by I-Wings-I**
## Summary

Added In The Create_Webhook For Text Channels To Forums And Voice Channels.

Weirdly on the voice channel, the webhook comes back good and does work, but it doesn't show in the Integration tab when editing the channel.

## Checklist

- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
  - [ ] I have run `task pyright` and fixed the relevant issues.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by toolifelesstocode — APPROVED** [review]

File: nextcord/channel.py [hunk 1]
```diff
@@ -1902,6 +1954,58 @@ def is_me(m):
...
+        name: str,
+        avatar: Optional[Union[bytes, Asset, Attachment, File]] = None,
+        reason: Optional[str] = None,
+    ) -> Webhook:
+        """|coro|
+
+        Creates a webhook for this channel.
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
+
```
Comment (EmreTech) [inline nextcord/channel.py:1969]: Missing `..versionadded`

File: nextcord/channel.py [hunk 2]
```diff
@@ -1411,6 +1411,58 @@ def archived_threads(
...
```
Comment (EmreTech) [inline nextcord/channel.py:1426]: Missing `..versionadded`

**Review by EmreTech — CHANGES_REQUESTED** [review]

File: nextcord/channel.py [hunk 3]
```diff
@@ -1411,6 +1411,53 @@ def archived_threads(
...
+        name: str,
+        avatar: Optional[Union[bytes, Asset, Attachment, File]] = None,
+        reason: Optional[str] = None,
+    ) -> Webhook:
+        """|coro|
+
+        ..versionadded:: 2.3.2
+            Creates a webhook for this channel.
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
```
Comment (EmreTech) [inline nextcord/channel.py:1426]:
```suggestion
        Requires :attr:`~.Permissions.manage_webhooks` permissions.

        ..versionadded:: 2.4
```

File: nextcord/channel.py [hunk 4]
```diff
@@ -1902,6 +1949,53 @@ def is_me(m):
...
```
Comment (EmreTech) [inline nextcord/channel.py:1964]:
```suggestion
        Requires :attr:`~.Permissions.manage_webhooks` permissions.

        ..versionadded:: 2.4
```

**Review by EmreTech — CHANGES_REQUESTED** [review]

File: nextcord/channel.py [hunk 5]
```diff
@@ -1411,6 +1411,52 @@ def archived_threads(
...
+    async def create_webhook(
+        self,
+        *,
+        name: str,
+        avatar: Optional[Union[bytes, Asset, Attachment, File]] = None,
+        reason: Optional[str] = None,
+    ) -> Webhook:
+        """|coro|
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
```
Comment (EmreTech) [inline nextcord/channel.py:1423]:
```suggestion
        Creates a webhook for this channel.

        Requires :attr:`~.Permissions.manage_webhooks` permissions.
```
The description got removed.

File: nextcord/channel.py [hunk 6]
```diff
@@ -1902,6 +1948,52 @@ def is_me(m):
...
```
Comment (EmreTech) [inline nextcord/channel.py:1960]:
```suggestion
        Creates a webhook for this channel.

        Requires :attr:`~.Permissions.manage_webhooks` permissions.
```

**Review by EmreTech — CHANGES_REQUESTED** [review]

**Review by toolifelesstocode — APPROVED** [review]
(Approving the new changes)

File: nextcord/channel.py [hunk 7]
```diff
@@ -1411,6 +1411,53 @@ def archived_threads(
...
 
+    async def create_webhook(
+        self,
+        *,
+        name: str,
+        avatar: Optional[Union[bytes, Asset, Attachment, File]] = None,
+        reason: Optional[str] = None,
+    ) -> Webhook:
+        """|coro|
+        Creates a webhook for this channel.
```
Comment (EmreTech) [inline nextcord/channel.py:1422]:
```suggestion
        """|coro|

        Creates a webhook for this channel.
```

File: nextcord/channel.py [hunk 8]
```diff
@@ -1902,6 +1949,53 @@ def is_me(m):
...
```
Comment (EmreTech) [inline nextcord/channel.py:1960]:
```suggestion
        """|coro|

        Creates a webhook for this channel.
```

File: nextcord/channel.py [hunk 9]
```diff
@@ -1411,6 +1411,53 @@ def archived_threads(
...
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
+
+        ..versionadded:: 2.4
```
Comment (ooliver1) [inline nextcord/channel.py:1426]:
```suggestion
        .. versionadded:: 2.4
```
image
Spacing matters.

File: nextcord/channel.py [hunk 10]
```diff
@@ -1902,6 +1949,53 @@ def is_me(m):
...
```
Comment (ooliver1) [inline nextcord/channel.py:1964]:
```suggestion
        .. versionadded:: 2.4
```

**Review by ooliver1 — CHANGES_REQUESTED** [review]

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**